x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exceptions_viewer_header.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiFieldSearch,
+  EuiButtonGroup,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiContextMenu,
+  EuiButton,
+  EuiContextMenuPanelDescriptor,
+  EuiButtonGroupOption,
+} from '@elastic/eui';
+import React, { useState, useCallback, useMemo } from 'react';
+
+import * as i18n from '../translations';
+import { ToggleId } from '../types';
+
+interface ExceptionsViewerHeaderProps {
+  selectedListType: ToggleId;
+  isInitLoading: boolean;
+  listTypes: ToggleId[];
+  onFiltersChange: ({ filter, tags }: { filter: string; tags: string[] }) => void;
+  onAddExceptionClick: (type: ToggleId) => void;
+  onToggleListType: (type: ToggleId) => void;
+}
+
+/**
+ * Collection of filters and toggles for filtering exception items.
+ */
+const ExceptionsViewerHeaderComponent = ({
+  selectedListType,
+  isInitLoading,
```

Comment:
@marrasherrier  -- how do we want to handle the loading state for exception lists? I know we're trying to get away from the panels around all our components, so perhaps just the default spinner we use for tables and make it flush with page background (+ min height to reduce jitter)? Then show the empty-view if there's nothing to show? Or do we want a sweet custom skeleton loader similar to the `<EuiLoadingContent lines={3} />` component??  😀 


<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/83820583-4fa2fa00-a68a-11ea-9725-abaa58895a60.gif" />
</p>

Reply:
Design followed up saying to use whichever. Your update to use the common loading spinner looks good to me! 👍